spacy/tests/pipeline/test_tagger.py

```diff
@@ -67,6 +67,29 @@ def test_tagger_initialize_tag_map():
 ]
 
 
+def test_label_smoothing():
```

Comment:
Maybe a backwards compatibility test?

* Create an `nlp` object
* Add a tagger
* Remove the `label_smoothing` entry from its config
* Call `to_bytes()`
* Load back into a new NLP object
* Make sure that succeeds